lnxrouter

```diff
@@ -179,8 +179,9 @@ define_global_variables(){
     WIFI_IFACE=
     CHANNEL=default 
     WPA_VERSION=2
-    MAC_FILTER=0
```

Comment:
Since we're changing the meaning of this variable (from "enable mac filter or not" to "macaddr_acl in hostapd.conf" ),
we should use a new variable name MACADDR_ACL